<launch>
    <node name="path" pkg="rviz" type="rviz" args="-d $(find path_generator)/config/path_generator.rviz"/>
    <arg name="vel" default="1.0"/>
    <arg name="yaw_rate" default="0.3"/>
    <arg name="arv_dist" default="0.1"/>
    <param name="LoS_radius" value="0.1"/>
    
    <param name="sitl_flag" value="True"/>
    <param name="file_dir" value="$(find path_generator)/path/indoor/path1"/>
    
    <group ns="uav4">
        <param name="robot_name" value="uav4"/>
        <param name="control_type" value="1"/><!-- 0: pursuit 1: LoS-->
        <param name="rot_type" value="1"/><!-- 0: angle 1: rate-->
        <param name="vel" value="$(arg vel)"/>
        <param name="yaw_rate" value="$(arg yaw_rate)"/>
        <param name="arv_dist" value="$(arg arv_dist)"/>
        <!-- control node -->
        <node pkg="path_generator" type="velocity_pursuit_guidance.py" name="control" output="screen"/>
        <node pkg="path_generator" type="gt_parse_vision.py" name="gt" output="screen"/>
    </group>
</launch>
